### PR TITLE
Fixes (#13) poster glitch fixed

### PR DESCRIPTION
**Pull request by stg2209**
I have changed the margin-top as per required. 
before
after

### CODE REVIEW

**Comment by stg2209** [comment]
And regarding configuration, I don't know maybe it was just in my case or what. But i think you should update the token and firebase configuration in readme. firstly the tmdb token format in readme file is not the correct one. All the tokens ( firebase & tmdb ) should be written in the env.template format. And also you can add in readme that the tmdb api key to be used should be of ' API Read Access Token '.

File: src/components/Movie_basic_details/style.scss
Comment (dipandhali2021) [inline src/components/Movie_basic_details/style.scss:1]: LGTM

File: package-lock.json
[the remark this answers is not in the document]
Comment (stg2209) [reply package-lock.json:1]: Done

File: package.json
[the remark this answers is not in the document]
Comment (stg2209) [reply package.json:1]: done

File: src/App.jsx
[the remark this answers is not in the document]
Comment (stg2209) [reply src/App.jsx:1]: Done

File: src/components/Movie_basic_details/style.scss
Comment (stg2209) [reply src/components/Movie_basic_details/style.scss:1]: 👍

**Review by dipandhali2021 — APPROVED** [review]
Good Work👏👏

**Pull request merged**